controllers/admin/AdminAdyenPrestashopCronController.php

```diff
@@ -41,6 +41,7 @@ class AdminAdyenPrestashopCronController extends \ModuleAdminController
     public function __construct()
     {
         $this->helperData = ServiceLocator::get('Adyen\PrestaShop\helper\Data');
+        $this->logger = ServiceLocator::get('Adyen\PrestaShop\service\Logger');
```

Comment:
This property is not declared in the class

Reply:
indeed I will add the helperData and the logger as a property and not just dynamically declaring them